packages/server/src/modules/common/i18n.js

```diff
@@ -0,0 +1,14 @@
+import i18n from 'i18n';
+
+export default (language = i18n.getLocale(), moduleName, messageKey) => {
+  const clientLanguage = language.split('-')[0]; // Supports both options 'en' and 'en-US'
+  const translation = i18n.getCatalog(clientLanguage)[moduleName][messageKey];
+
+  if (!translation) {
```

Comment:
Exception shouldn't be thrown here, if there is no translation, the translation for fallbackLng should be used, as it is done for client-side i18n. Generally you should study carefully client i18n and make server i18n work as close to it as possible.